apis/v1/jaeger_types.go

```diff
@@ -276,6 +276,9 @@ type JaegerQuerySpec struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Strategy"
 	Strategy *appsv1.DeploymentStrategy `json:"strategy,omitempty"`
+
+	// +optional
+	UUID string `json:"uuid,omitempty"`
```

Comment:
Would it make sense to add the uuid to the [JaegerCommonSpec](https://github.com/luohua13/jaeger-operator/blob/1b161187aa7164b793c421ee1441b64bd141cd2f/apis/v1/jaeger_types.go#L192)?

Reply:
Yeah, you are right! Then I will close the pr.